Portals/_default/Containers/Vanjaro/Base.ascx.cs

```diff
@@ -36,8 +37,12 @@ protected void Page_Load(object sender, EventArgs e)
                                     url += "&skinsrc=" + "[g]skins/vanjaro/base";
                                 else
                                     url += "?skinsrc=" + "[g]skins/vanjaro/base";
-                                if (splitarray.Length > 1)
-                                    url += "#!/" + splitarray[1];
+
+                                string TempSplitArray = splitarray.Length > 1 ? splitarray[1] : string.Empty;
+                                if (splitarray.Length > 1 && TempSplitArray.StartsWith("/"))
```

Comment:
Please remove `splitarray.Length > 1` and add instead of this `!string.IsNullOrEmpty(TempSplitArray)`